Utils/StringConversions.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2017 MapD Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _STRING_CONVERSIONS_H_
+#define _STRING_CONVERSIONS_H_
+#include <boost/utility/string_view.hpp>
+#include <string>
+
+namespace StringConversions {
+
+const inline std::string& to_string(const std::string& s) {
+  return s;
+}
+
+std::string inline to_string(const boost::string_view& s) {
+  return s.to_string();
+}
+
+template <typename String>
+int64_t strtol(const String& s) {
```

Comment:
I prefer `std::stoll(s.to_string())` instead of writing the function from scratch.

Reply:
It is one of the main optimizations here. `std::stoll(to_string(s))` will cause copying, which is unnecessary here. I could use c++17 `std::to_chars` since you are moving to c++17. But I'm not sure how it implemented. Will need some benchmarking here.